modules/navier_stokes/include/linearfvkernels/LinearFVDivergence.h

```diff
@@ -0,0 +1,50 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "LinearFVFluxKernel.h"
+
+/**
+ * Kernel that adds contributions from an advection term discretized using the finite volume method
```

Comment:
```suggestion
 * Kernel that adds contributions from the mass advection term discretized using the finite volume method
```
do you foresee re-using this kernel for scalars?

Reply:
There is no mass in the formulation, this is just a face flux. Can be scalar flux, can be mass flux. I'd like to keep this general for now.